modules/core/04-channel/v2/keeper/keeper.go

```diff
@@ -75,17 +75,17 @@ func (k *Keeper) GetCounterparty(ctx context.Context, clientID string) (types.Co
 }
 
 // GetPacketReceipt returns the packet receipt from the packet receipt path based on the sourceID and sequence.
-func (k *Keeper) GetPacketReceipt(ctx context.Context, sourceID string, sequence uint64) (string, bool) {
+func (k *Keeper) GetPacketReceipt(ctx context.Context, sourceID string, sequence uint64) []byte {
```

Comment:
why the removal of the `bool` return value?

Reply:
Looking at the usage later, I guess technically i provides no added value as we can do a len check in line, and also we would never be storing an empty bz array so that case does not need to be differentiated. I also see we do it this way in the channel v1 stuff, will leave the comment here in case anyone else was wondering!

Reply:
Yeah, might change this back to be honest. As v1 has the bool return value for receipts (but not commitments - which I also changed). 

Actually had a todo which I forgot to address here where the usage should be `HasReceipt` I think. I can push an update before merging!